controllers/adverse-action.js

```diff
@@ -0,0 +1,51 @@
+const AdverseAction = require("../models/adverse-action");
+const Candidate = require("../models/candidate");
+
+exports.getAdverseActions = (req, res, next) => {
+  AdverseAction.find()
+    .populate("candidate", "name")
+    .exec()
+    .then((result) => {
+      return res.status(200).json({
+        message: "Fetched Adverse Actions successfully.",
+        adverseActions: result,
```

Comment:
instead of adverseActions, use the keyname "data" for all the responses so that it is common for all the APIs.